torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -3360,370 +3360,372 @@ struct to_ir {
   }
 };
 
-CompilationUnit::CompilationUnit(const std::string& source)
-    : CompilationUnit() {
-  // calles the define with native resolver to generate the graph for functions
-  define(c10::nullopt, source, nativeResolver(), nullptr);
-}
+//CompilationUnit::CompilationUnit(const std::string& source)
```

Comment:
Should we delete code instead of commenting them out?

Reply:
Oops, thanks for catching this!